### PR TITLE
Mathbox: add quantitative versions of some set theory existence theor…

### PR DESCRIPTION
**Pull request by benjub**
…ems (and edit one commit in Main accordingly).

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -50217,9 +50217,10 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
   univ $p |- U. _V = _V $=
     ( cvv cpw cuni pwv unieqi unipw eqtr3i ) ABZCACAHADEAFG $.
 
-  $( Membership of a power class.  Exercise 10 of [Enderton] p. 26.
-     (Contributed by NM, 13-Jan-2007.) $)
-  pwel $p |- ( A e. B -> ~P A e. ~P ~P U. B ) $=
+  $( Quantitative version of ~ pwexg : the powerset of an element of a class is
+     an element of the double powerclass of the union of that class.  Exercise
+     10 of [Enderton] p. 26.  (Contributed by NM, 13-Jan-2007.) $)
+  pwel $p |- ( A e. V -> ~P A e. ~P ~P U. V ) $=
```
Comment (avekens) [inline set.mm:50223]: Why did you change ` B ` to ` V `? We usually use ` V ` for an arbitrary class, especially in the idioms ` A e. V ` to express "`A ` is a set". In this case, however, ` V `is not arbitrary, because it appears at the RHS of the implication ("that class").  Therefore, I thinlk ` B ` is a better choice here.

File: set.mm [hunk 2]
```diff
@@ -538886,6 +538891,29 @@ is a Moore collection ( ~ bj-snmoore ), this can also be stated as
       VAUTMFDRWDVBVCVDVEVF $.
   $}
 
+  ${
+    $d A x $.
+    $( A pair formed of the empty set and any class is a Moore collection (on
+       ` U. { (/) , A } ` , which is ` A ` if it is a set, else ` (/) ` ).
+
+       One also has ` |- ( ( A C_ B /\ A e. V ) -> { A , B } e. Moore_ ) ` ,
```
Comment (avekens) [inline set.mm:538899]: Do you plan to provide this as theorem soon?

Comment (benjub) [reply set.mm:538899]: Yes, it's only a small variant of that theorem (basically, replace in the proof the line $\varnothing \cap A= \varnothing$ by $A \subseteq B \to A \cap B = A$), but I won't have time in this MR.

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:50223]: Done.

**Review by avekens — APPROVED** [review]

File: set.mm (hunk 2 above)
Comment (tirix) [reply set.mm:538899]: Wouldn't it be easier to prove the empty set version from the subset version?
I.e. to prove that coming theorem from this version?

Comment (benjub) [reply set.mm:538899]: Of course: once I prove the general theorem, the one with the empty set will be a direct consequence (as will be the forward implication in `|- ( A e. _V <-> { A } e. Moore_ )`).
